recipe.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+            recipe.htm
+        </title>
+    </head>
+        <body>
+            <h1>
+            <strong>The Perfect PB&J</strong>
+            </h1>
+                <img src="https://www.landolakes.com/RecipeManagementSystem/media/Recipe-Media-Files/Recipes/Retail/DesktopImages/2018_Grilled-Peanut-Butter-and-Jelly_20336_600x600.jpg?ext=.jpg"/>
```

Comment:
Get used to adding `alt='insert a description of the image here ' ` on your `<img> ` tags, later on they will throw errors if you don't have them.